zipline/data/data_portal.py

```diff
@@ -157,6 +158,13 @@ def __init__(self,
         # days data starts.
         self._day_offsets = None
 
+    @property
+    def minute_bar_reader(self):
```

Comment:
The reason I hadn't it is because it is lazily created (i.e. not created until first use, because some tests/use cases were not writing minute data), and the `_minute_bar_reader` is used as the placeholder/None value.